ppa/common/admin.py

```diff
@@ -13,5 +13,10 @@ def group_names(self, obj):
     group_names.short_description = 'groups'
 
 
+class CollapsibleTabularInline(admin.TabularInline):
```

Comment:
Is this used somewhere?

Reply:
It isn't. It's from my first draft that used inlines. I will remove it when finalizing work on this pull request's revisions.